### ISSUE TITLE
Last release looks broken

### ISSUE THREAD

**Issue by posva** [znck/rollup-plugin-vue#37]
Hi again, the last release seems to be broken. When installing with yarn you get:

```
error Couldn't find a package.json (or bower.json) file in /Users/posva/.yarn-cache/npm-rollup-plugin-vue-2.2.9
```

I searched but couldn't find any useful information. The package.json is there and removing node modules + cleaning the cache does nothing. 😔

**Comment by znck** [comment]
I have to test it once (Used yarn first time to publish a release).

**Comment by znck** [comment]
Fixed with v2.2.10
